File: frontend/CarRental/src/app/models/cars-get-dto.ts
export interface CarsGetDto {
    id: number;
    licenseplate: string;
    brand: string;
    model: string;
    year: number;
    kilometrage: number;
    rentPrice: number;
    CarStatus: CarStatus;
}

export enum CarStatus {
  Available,
  Unavailable,
  Service 
}

File: backend/DTO/CarDTO/CarsGetDTO.cs
namespace CarRentalSystem;

public class CarsGetDTO
{
    public int Id {get; set;}
    public string LicensePlate {get; set;}
    public string Brand {get; set;}
    public string Model {get; set;}
    public int Year {get; set;}
    public int Kilometrage {get; set;}
    public int RentPrice {get; set;}
    public CarStatus CarStatus {get; set;}
}